extensions/ql-vscode/src/remote-queries/run-remote-query.ts

```diff
@@ -280,6 +280,8 @@ export async function runRemoteQuery(
 
       variantAnalysisManager.onVariantAnalysisSubmitted(processedVariantAnalysis);
 
+      await variantAnalysisManager.prepareStorageDirectory(processedVariantAnalysis.id);
```

Comment:
I wasn't sure where to do this 🤔 Just after the variant analysis is submitted seemed like a good place, but I'm open to suggestions!

Reply:
I think that's a good time to do this. However, I think it'd be nicer if we do inside `variantAnalysisManager.onVariantAnalysisSubmitted()`

Reply:
Done in https://github.com/github/vscode-codeql/pull/1631/commits/6b7ebf543c903da865b84a03fd724a2fb0d5e886!